db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js

```diff
@@ -627,7 +690,8 @@ VALUES
 		NOKIA_4G_KEY_PARAMAETERS : NOKIA_4G_KEY_PARAMAETERS,
 		NETWORK_CELLS : NETWORK_CELLS,
 		NETWORK_SITES : NETWORK_SITES,
-		NETWORK_NODES : NETWORK_NODES
+		NETWORK_NODES : NETWORK_NODES,
+		NETWORK_3G3G_Relations : NETWORK_3G3G_Relations
```

Comment:
Change **_Relations** to **_RELATIONS** for consistency with the other query names .i.e. keep it in uppercase.